Makefile

```diff
@@ -35,6 +35,7 @@ build: site/out/index.html $(shell find . -not -path './vendor/*' -type f -name
 	# build slim artifacts and copy them to the site output directory
 	./scripts/build_go_slim.sh \
 		--version "$(VERSION)" \
+		--compress 22 \
```

Comment:
The release scripts don't call the Makefile directly for binary builds because make tries to regenerate every single file, so make sure you copy this to the `coder.yaml` (under `deploy`) and `release.yaml` (under `linux-windows` and `darwin`) workflows

Reply:
What do you think about setting this one to `6` for faster dev builds? Or is there a risk we refactor and end up with mixed compression levels?